projects/fsastorefrontlib/src/cms-components/my-account/order/order.module.ts

```diff
@@ -42,9 +47,15 @@ const moduleComponents = [
       cmsComponents: {
         AccountOrderDetailsItemsComponent: {
           component: FSOrderDetailItemsComponent,
+          guards: [AuthGuard],
         },
         AccountOrderDetailsTotalsComponent: {
           component: FSOrderDetailTotalsComponent,
+          guards: [AuthGuard],
+        },
+        AccountOrderHistoryComponent: {
+          component: FSOrderHistoryComponent,
+          guards: [AuthGuard],
```

Comment:
Is guard needed? All page routes are already guarded by CMSPageGuard & AuthGuard